src/Syntax.hs

```diff
@@ -1,8 +1,17 @@
+{-|
+Module           : Syntax
+Description      : User-facing syntax of Klister
+
+'Syntax' is the result of parsing Klister code. It is transformed into
+Klister\'s core language by the expander.
```

Comment:
This is true, but incomplete. Some Syntax also results from the expansion of user macros.

I'd instead say that this is the user-facing syntax for Klister, and that it can result either from parsing user code or from steps of macro expansion.